ci.sh

```diff
@@ -7,14 +7,16 @@ PROJECT_DIR="$DIR"
 
 pushd "$PROJECT_DIR"
 
-# For some reason test for annotation processor are failing on a regular CI setup.
-# So we had to exclude test task for it from the main build process and execute it as a separate command.
-./gradlew clean build checkstyle -PdisablePreDex --exclude-task :storio-sqlite-annotations-processor-test:test --exclude-task :storio-content-resolver-annotations-processor-test:test
-./gradlew :storio-sqlite-annotations-processor-test:test
-./gradlew :storio-content-resolver-annotations-processor-test:test
-
 # Export "PUBLISH_RELEASE=true" to initiate release process.
-if [ "$PUBLISH_RELEASE" == "true" ]; then
+if [ "$PUBLISH_RELEASE" != "true" ]; then
+    echo "Running non-release build...".
```

Comment:
Additionally I decided to wrap non-release build in if statement as well, otherwise it was always running repeating unnecessary build parts for publish state